### ISSUE TITLE
tkn taskrun describe, doesn't show any sidecar info

### ISSUE THREAD

**Issue by bitsofinfo** [tektoncd/pipeline#1968]
# Expected Behavior

`tkn taskrun describe` would show sidecar info/basic status

# Actual Behavior

doesn't show

**Comment by dibyom** [comment]
This might better belong in the CLI repo I think?
@chmouel @vdemeester

**Comment by danielhelfand** [comment]
Yes, if you could open this in the [cli repo issues](https://github.com/tektoncd/cli/issues), it would be a better place to track and discuss, @bitsofinfo. 

But yes, `tkn tr desc` does not have sidecar information at this time, and I think it would be a good feature to pursue.

**Comment by chmouel** [comment]
I think @vdemeester have the rights to transfer it in there! 

/summon @vdemeester
